Dockerfile

```diff
@@ -1,17 +1,27 @@
 # Build the binary
-FROM golang:1.14.1 as builder
+FROM golang:1.14.12 as builder
 
 WORKDIR /workspace
+
+# Copy the Go Modules manifests
+COPY go.mod .
+COPY go.sum .
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
 # Copy the go source
-COPY . .
+COPY main.go .
+COPY api/ api/
+COPY pkg/ pkg/
```

Comment:
@wtam2018 I restored these lines because of the way Docker copies directories. `COPY api/ .`, for example, copies the contents of `api/` to the root of the workdir. We don't want that.

Reply:
I thought that it was your intent to copy the entire folder to the root of WORKDIR.    It's fine to restore it.